caffe2/core/export_caffe2_op_to_c10.h

```diff
@@ -165,6 +165,18 @@ inline FunctionSchema make_function_schema_for_c10(const char* schema_str) {
   }                                                                \
   }
 
+#define C10_EXPORT_CAFFE2_OP_TO_C10_SCHEMA_ONLY(OperatorName, OperatorSchema)\
+  /* Register the op schema with the c10 dispatcher */                       \
+  namespace caffe2 {                                                         \
+  namespace _c10_ops {                                                       \
+  C10_EXPORT const FunctionSchema& schema_##OperatorName() {                 \
+    static const FunctionSchema schema =                                     \
+        ::caffe2::detail::make_function_schema_for_c10(OperatorSchema);      \
+    return schema;                                                           \
+  }                                                                          \
+  }                                                                          \
+  }
+
 #define C10_EXPORT_CAFFE2_OP_TO_C10_CPU(                                     \
```

Comment:
Can you rewrite this one as calls to`_SCHEMA_ONLY` and `_KERNEL_ONLY` to avoid the duplication?